Replaced `async void` with `async Task`

trailmax opened the pull request:
As per MS recommendation, it should be avoided using `async void` as it hides exceptions

azurecla commented:
Hi __@jeannt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jeannt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

trailmax committed:
Replaced `async void` with `async Task`

As per MS recommendation, it should be avoided using `async void` as it hides exceptions
